tests/test_suite.py

```diff
@@ -211,14 +220,32 @@ def parse_args():
                                  dest="scenarios_regex",
                                  default=None,
                                  help="Regular expression matching test scenarios to run")
+    parser_combined.add_argument("--slice",
```

Comment:
What about using https://docs.python.org/3/library/argparse.html#metavar ?
Right now this is how the help message looks like:
```
  --slice _SLICES _SLICES
                        Allows to run only Xth slice of Y in total, to enable stable parallelization
                        of the bigger test sets.
```
which is not that nice. With added metavar (e.g. `metavar=('X', 'Y')`) it would be:
```
  --slice X Y
```
Also `Note that metavar only changes the displayed name - the name of the attribute on the parse_args() object is still determined by the dest value`.

Reply:
*Facepalm* yeah, I believe I was searching for it, but for whatever reason missed the `metavar`.. Thanks, forcepushed new version with proper help output :)